src/assets/config.json

```diff
@@ -25,6 +25,8 @@
   "money": 10000,
   "satisfaction": 100,
 
+  "penaltySendingCustomerAway": 150,
```

Comment:
Are we sure we want to do this? Perhaps set it to `0` for now, and bring it up tomorrow in a discussion?